app/controllers/transferDetails/IsTransferCashOnlyController.scala

```diff
@@ -60,9 +65,31 @@ class IsTransferCashOnlyController @Inject() (
           Future.successful(BadRequest(view(formWithErrors, mode))),
         value =>
           for {
-            updatedAnswers <- Future.fromTry(request.userAnswers.set(IsTransferCashOnlyPage, value))
+            baseAnswers    <- Future.fromTry(request.userAnswers.set(IsTransferCashOnlyPage, value)) // set cash only to true
+            updatedAnswers <- if (value) {
```

Comment:
This seems a bit unwieldy, maybe consider pulling the if statement out into a small helper?